hedera-node/src/test/java/com/hedera/test/utils/ConverterUtils.java

```diff
@@ -0,0 +1,62 @@
+package com.hedera.test.utils;
+
+/*-
+ * ‌
+ * Hedera Services Node
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import org.junit.jupiter.params.converter.ArgumentConversionException;
+
+/**
+ * Contians various common checks and methods used by the Converter classes
+ * @author abhishekpandey
+ * */
+public final class ConverterUtils {
+    /**
+     * Checks if the input to the converter is of type {@link String} else throws an {@link ArgumentConversionException}
+     * @param input
+     *              the input to the converter
+     * @throws ArgumentConversionException
```

Comment:
nit: @throws ArgumentConversionException when the input is not of type String

Reply:
Fixed, thanks!